pkgs/by-name/he/hellmaker/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hell-maker";
```

```suggestion
  pname = "hellmaker";
```
This needs to be equal to top-level name. In this case `hellmaker`.